packages/radix-ui-themes/src/components/radio-card-group.css

```diff
@@ -0,0 +1,138 @@
+      --radio-card-group-item-padding: var(--space-4);
+      --radio-card-group-item-radius: var(--radius-4);
+      --radio-card-group-item-gap: var(--space-3);
+    }
+    &:where(.rt-r-size-3) {
+      --radio-card-group-item-padding: var(--space-5);
+      --radio-card-group-item-radius: var(--radius-5);
+      --radio-card-group-item-gap: var(--space-4);
+    }
+  }
+}
+
+/***************************************************************************************************
+ *                                                                                                 *
+ * VARIANTS                                                                                        *
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+.rt-RadioCardGroupItem {
+  position: relative;
+  /* pseudo element for focus outline */
+  &::before {
+    content: '';
+    display: block;
+    position: absolute;
+    inset: 0;
+    border-radius: inherit;
+  }
+}
+
+/* surface */
+
+.rt-RadioCardGroupRoot:where(.rt-variant-surface) {
+  :where(.rt-RadioCardGroupItem) {
+    background-color: var(--color-surface);
+    box-shadow: inset 0 0 0 1px var(--gray-a5);
+
+    &:where(:hover) {
+      box-shadow: inset 0 0 0 1px var(--gray-a7);
+    }
+
+    &:where([data-state='checked']) {
+      outline: 2px solid var(--accent-10);
+      outline-offset: -2px;
+    }
+
+    &:where(:focus-visible)::before {
+      outline: 2px solid var(--color-focus-root);
+      outline-offset: 2px;
+    }
+  }
+
+  &:where(.rt-high-contrast) :where(.rt-RadioCardGroupItem) {
+    &:where([data-state='checked']) {
+      outline-color: var(--accent-12);
+    }
+    &:where(:focus-visible) {
+      outline-color: var(--accent-12);
+    }
+  }
+}
+
+/* classic */
+
+.rt-RadioCardGroupRoot:where(.rt-variant-classic) {
+  :where(.rt-RadioCardGroupItem) {
+    background-color: var(--color-surface);
+    /* A transparent shadow to transition smoothly to `:active` */
+    box-shadow: 0 0 0 1px var(--color-transparent), var(--shadow-2);
+    transition: box-shadow 120ms;
+
+    &:where(:hover) {
+      transition-duration: 40ms;
+      box-shadow: var(--card-classic-hover-box-shadow);
+    }
+
+    &:where(:active) {
+      transition-duration: 40ms;
+      box-shadow: 0 0 0 1px var(--gray-a5), var(--shadow-2);
+    }
+
+    &:where([data-state='checked']) {
+      outline: 2px solid var(--accent-9);
+      outline-offset: -1px;
+    }
+
+    &:where(:focus-visible)::before {
+      outline: 2px solid var(--color-focus-root);
+      outline-offset: 3px;
+    }
+  }
+}
+
+/* all high-contrast */
+
+.rt-RadioCardGroupRoot:where(.rt-high-contrast) :where(.rt-RadioCardGroupItem) {
+  &:where([data-state='checked']) {
+    outline-color: var(--accent-12);
+  }
+  &:where(:focus-visible) {
+    outline-color: var(--accent-12);
+  }
```

Comment:
This doesn't work, but we don't really need it—don't think we adjust the focus outline color on the high contrast variants anymore

Reply:
Yep, that's a mistake, wasn't trying to do that anyway as we don't adjust it like you say.